composer.json

```diff
@@ -33,7 +33,8 @@
         "doctrine/lexer": "^2.1 || ^3",
         "doctrine/persistence": "^3.1.1",
         "psr/cache": "^1 || ^2 || ^3",
-        "symfony/console": "^5.4 || ^6.0 || ^7.0"
+        "symfony/console": "^5.4 || ^6.0 || ^7.0",
+        "symfony/var-exporter": "^5.4 || ^6.2"
```

Comment:
```suggestion
        "symfony/var-exporter": "^6.2"
```

Lazy ghost objects don't exist in v5.4. We need version 6.

Reply:
:thinking: this makes me notice that for some reason, we don't have a job with `--prefer-lowest` on this branch.

Reply:
@derrabus apparently you removed it in https://github.com/doctrine/orm/pull/9069… do you recall why?

Reply:
No, let's add it back.

Reply:
Done in #10971